<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>discussion</title>
</head>
<body>
   <h1>Discussion</h1>
   <p>In the present study we first report the high-resolution linkage map for European aspen (P. tremula). This is the most widespread Populus species in the Northern hemisphere and harbors substantial genetic diversity both within populations and across the species range. Levels of nucleotide polymorphism for many nuclear genes of P. tremula were reported to be 2- to 10-fold higher than those in other woody, long-lived perennial plants, such as Pinus and Cryptomeria, and 2 to 5-fold higher than those in Betula pendula. Remarkably, the estimated population subdivision is low, thus, a substantial fraction of the aspen species-wide diversity can be sampled from single populations [28].</p>
   <p>Due to the rapid decay of LD in P. tremula, it was previosuly assumed that the number of markers needed to ensure adequate genome coverage for QTL mapping purposes may be too high to be feasible in this species. On other hand, the rapid decay of LD suggests that it may be possible to map functional variation to very fine scales in P. tremula [28]. The limitation in the number of markers was overcome with the development of NGS technologies (e.g. RADseq [29]), so currently one could take the advantage of the low LD in aspen to achieve fine mapping and candidate genes identification for P. tremula.</p>
   <p>There are numerous potential targets for the QTL fine mapping in aspen. The huge natural intraspecific genetic variation of P. tremula is reflected in the phenotypic diversity and constantly attracts the attention of forest researchers. At least 10 P. tremula varieties have been described in the last century just by the shape of leaves (var. monticola, parvifolia, grandifolia, rotundifolia, minor, oxyodonta, stricta, laevigata, pendula, supine) and two varieties have been reported based on bark color difference: the very ordinary light-colored P. tremula var. vulgaris and the dark-colored P. tremula var. fisca that occurs occasionally in the middle and southern regions of Russia [30]. Phenological races were also designated as intraspecific taxa: aspen with late bud flush (P. tremula L. var. tardifolia N. Rubtz.) was described from Trans-Ili Alatau (Tien Shan) [31] and reported for Belarus [32]. The late bud flushing variety showed 10–15 days delay compared to an ordinary early aspen. Revising the aspen intra-specific diversity A.Tzarev [3] reported 3% of occurrence of the late bud flushing variety in the forest-steppe regions of Russia (Voronezh area). There, bud flush delay varied from 6 to 14 days depending on the weather conditions in spring, with the late form requiring effective temperatures 1.7–2-fold higher than the early form of aspen. Remarkably, the inspection of plantations of early and late-bud flushing varieties over 30 years old showed that the percentage of trees affected by heart rot is less in plantations of the late form in approximately 1.5 times than in plantations of early aspen [3]. Special attention was traditionally paid to aspen clones with natural resistance to heart rot. Beginning from the nineteenth century small fragmentary natural populations of healthy aspen trees were regularly described within forests dramatically damaged by Phellinus pathogen in the Central part of Russia. Similarly, groups of 50–60 trees in otherwise infected aspen forests of Northwest Russia produced coppice shoots at late age (80–90 years), with no any visible signs of fungi destruction. It was intuitively concluded that resistance to the Phellinus pathogen in aspen is an inherited trait suitable for breeding [1].</p>
   <p>In this paper we report a set of 15,732 SNP markers that are polymorphic between two genotypes of P. tremula. This is only a subset of the nucleotide polymorphism in aspen and does not include those regions in P.tremula without homology to the P. trichocarpa reference genome. About 20% of reads were not mapped to the reference, thus, part of the polymorphism in aspen is still left unexplored. 30% of the SNPs that mapped to the P. trichocarpa reference genome produced heterozygous genotypes in the maternal and paternal aspen trees assessed in this study. This is in good concordance with previous reports showing 29.2% of loci in P. trichocarpa accessions carrying heterozygous genotypes [7]. The developed set of 15,732 SNP markers would contribute to further association mapping studies in aspen.</p>
   <p>Two thousand fifty five SNP markers were successfully employed for the construction of high-resolution maternal and paternal linkage maps for an F1 full-siblings P. tremula population. The constructed map comprised 19 linkage groups based on 1055 SNPs for the male map and 1000 SNPs for the female map. The estimated lengths of 3054.99 and 3090.56 cM for the two parental genetic maps were slightly shorter in comparison with the total genetic lengths reported earlier for the high density map of P. deltoides (4057 cM) and P. simonii (3.809 cM) [11]. On other hand, these values were larger than those reported in P. adenopoda (2443.2 cM) and P. alba (2719.5) [27] based on SSR and SRAP markers. The genetic lengths of the aspen linkage groups in our study were, however, consistent with the chromosome size of P. trichocarpa (Tables 1, 2).</p>
   <p>Despite the highly conserved synteny and collinearity observed between SNP order on the chromosomes of P. tremula and P. trichocarpa, our results suggest the presence of several chromosome inversions between the two taxonomically remoted Populus species. P. tremula belongs to the section Populus (syn. Leuce Duby), whereas balsam poplars P. trichocarpa and P. simonii are from the same section Tacamahaca Spach. Even for P. simonii genetic maps, several linkage groups exhibited inverse orders of some SNPs relative to the P. trichocarpa genome [11]. Moreover, for nearly all linkage groups of P. deltoides and P. simonii one or more local regions have been reported where SNP order was inconsistent with reference genome positions [12]. We hypothesize that for some linkage groups of P. tremula (e.g. Chr 1, 11, 15, 18) chromosome inversions could be exposed relatively to the SNP marker order on P. trichocarpa chromosomes. An inversion occurs when a chromosome breaks at two points and the segment bounded by the breakpoints is reinserted in the reversed orientation [33]. A key evolutionary effect of inversions is that they suppress recombination as heterozygotes. As a result, inversions may contribute to reproductive isolation, local adaptation and speciation in plants [34]. The comparison of the whole genome sequencing of P. tremula with the reference genome of P. trichocarpa would make it possible to determine whether the observed inconsistency in the order of SNP markers on two genomes is due to the inversion of chromosomes, rather than the distortion of the genetic map.</p>
</body>
</html>
