<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>methods</title>
</head>
<body>
   <h1>Methods</h1>
   <h3>Mapping population</h3>
   <p>To construct a high resolution linkage map an F1 full-sib mapping population was developed from the intra-specific cross of two aspen genotypes (P. tremula). Both male and female trees grew in the same location in the St. Petersburg area at a distance of only 8 m from each other. The parental trees had comparable ages: 30–32 years for the female and 28–30 years for the male tree, based on the number of the annual rings estimated by taking a core sample. The height of the female tree was h = 19.8 m and the diameter at breast height was d = 0.27 m. For the male tree those measurements were estimated as h = 17.6 m and d = 0.23 m, correspondingly. Inhabiting the same environment, both parents had similar dendrometry records, except for phenological traits and winter hardiness. Ten and 6 days delay were recorded for the female tree compared to the male tree for generative bud flush and vegetative bud flush, respectively. As a consequence, serious frost damage of twigs was registered  or the male parent, whereas no frost damage was recorded for the female tree. Controlled crossing was performed in April 2016 in a greenhouse of St. Petersburg State Forest Technical University. Freshly cut twigs of male and female aspen about 1 m long were placed in a 10-l clean plastic jug with water. After 10 days of incubation at room temperature (20 °C), catkins emergence was observed for both male and female twigs. Hybridization was conducted by artificial pollination of female flowers using a small brush with pollen collected from male catkins. Next, twigs with pollinated female catkins were bagged to exclude random airborne pollen. After 14 days female catkins produced small fruit splitting to release lots of tiny, cottony seeds. The seeds were immediately planted in small plastic containers (12x10x7cm) filled with wet peat substrate. Germination occurred within a couple of days after the planting (Additional file 1: Figure S1). After 2 weeks the seedlings were transferred int individual containers and placed in lining-out nursery (Additional file 1: Figure S2). In May 2016, the seedlings reached the height of approx. 50 cm and were re-planted into larger individual containers (9x9x11 cm). By September 2016 about 1100 containerized seedlings were ready for the field. 
The seedling stock was then divided into three parts to establish the F1 full-sibling aspen plantations in three diverse environments. 192 seedlings were planted in September 2016 in a field 70 km southwest of Saint Petersburg (59°16′33″N, 30°08′32″E). This location is classified as a humid continental climate; distinct moderating influence of the Baltic Sea cyclones result in warm, humid and short summers and long, moderately cold wet winters. Another set of 192 seedlings was transported to West Siberia and planted in a field nursery in Surgut (61°15′00″N, 73°25′00″E). This environment features a continental subarctic climate with long and very cold winters (daily mean temperature in January is (20 °C) and short but relatively warm summers. The third part of the containerized seedlings was used to establish the hybrid aspen plantation in the South of Russia (Voronezh, 51°40′19″N, 39°11′03″E). The Saint Petersburg population of F1 hybrids was used to construct the high-density linkage map.</p>
   <h3>DNA isolation</h3>
   <p>DNA was extracted from dried leaves collected from parental trees and each of 122 one-year progeny seedlings. For each sample 0.15 g of dried leaf material were frozen in liquid nitrogen and grinded with Mixer Mill MM 400 (Retsch, Germany). Grinded samples were washed with fresh washing buffer: 100 mM Tris-HCl, pH 8.0; 50 mM EDTA; 1 M NaCl; 1% 2-mercaptoethanol; 1% polyvinylpyrrolidone (PVP) (k-30, SABC) [16]. DNA extraction was carried out using a DNA Extran-3 kit (Syntol, Moscow) according to the manufacture protocol. Extracted DNA was cleaned with Genomic DNA Clean & Concentrator kit (Zymo Research)</p>
   <h3>Construction and sequencing of ddRAD libraries</h3>
   <p>Six ten double-digested RADseq (ddRADseq) libraries were constructed for the two parental genotypes (8 library replicates for each parent) and their 122 progenies (1 library per individual) according to the protocol described by Plekhanova et al. [17]. Concentration of DNA for each individual sample was normalized to 10 ng/ul and digested for 1 h at 37°C with 0.7 units of HindIII (NEB, USA) in a 20-ul reaction volume. Digestion was stopped by incubating the reaction mixture at 65°C for 20 min. Next, the “barcode” adapter was ligated to the end generated by HindIII, allowing pooling of the samples. The ligation reaction was carried out at 22°C for 1 h in a volume of 50 ul with 1.6 units of T4 ligase (NEB, USA). T4 ligase inactivation was reached by heating at 65°C for 10 min. Next, 10 ul for each sample were pooled and simultaneously purified with an Agencourt AMPure XP kit. The second restriction digestion was performed with 0.7 units of NlaIII (NEB, USA) in a 20-ul reaction volume at 37°C for 1 h with subsequent inactivation at 65°C for 20 min. The second “common” adapter was ligated to the overhanging end of NlaIII. The reaction mixture volume (50 ul) was dispersed into 14 aliquots and PCR was performed on the 14 aliquots to amplify the DNA fragments. PCR mix (50 ul) contained 1× HF buffer, 25 uM of each primer, 12.5 uM dNTPs and 1 unit of Phusion® High-Fidelity DNA Polymerase (NEB, USA). PCR conditions were: 98°C for 30 s, 14 cycles of 98°C for 10 s, 65°C for 30 s, 72°C for 15 s, then 72°C for 2 min. All 14 PCR reactions were pooled and purified with Agencourt AMPure XP kit. Concentration and size distribution of the prepared libraries were checked by automated electrophoresis with 2100 Bioanalyzer (Agilent Technologies, USA). Fragments were sequenced as 150 base reads on an Illumina HiSeq2500 at the University of Southern California Genome and Cytometry Core. Illumina data is available at NCBI under the BioProject PRJNA395596.</p>
   <h3>SNP calling</h3>
   <p>Illumina reads were mapped to the P. trichocarpa reference genome [6] using BWA MEM [18] under default mapping parameters. Polymorphisms were called using the GATK pipeline [19] which considers indel realignment and base quality score recalibration, and calls variants across all samples simultaneously through the HaplotypeCaller program in GATK. Variants were filtered using standard hard filtering parameters according to GATK Best Practices recommendation [20, 21]. More precisely, GBS data was filtered to only retain SNP calls with Mapping Quality (MQ) > 37 and Quality by Depth (QD) > 24. Both metrics take into consideration the quality of the mapping and genotype calls to ensure that only those with highest confidence were used. The SNPs were also filtered to retain those with MQRankSum < |2.0|, which ensure that there was no difference in the Mapping Quality scores for both alleles. This filtering retained those that pass all three criteria. Lastly, VCFtools software [22] was used to implement the following inclusion criteria: minor allele frequency (MAF) more than 3%, genotype call-rate more than 90%, and Hardy-Weinberg Equilibrium (HWE) exact P-value more than 10−5.</p>
   <h3>Linkage map construction</h3>
   <p>A RADseq linkage map of aspen was constructed using the pseudo-testcross mapping strategy [23], where a mapping population is developed by hybridizing two unrelated highly heterozygous parents to produce a set of F1 progeny [15]. With this strategy the maternal and paternal linkage maps were constructed separately using linkage phase inferred between adjacent markers. A linkage map was constructed using JoinMap 3.0 [24]. The linkage group assignments were made under the logarithm of odds (LOD) score limit of 5.0. The regression mapping algorithm and Kosambi’s mapping function were used for map construction with the following settings: Rec = 0.4, LOD = 1.0, Jump = 5. The resulting linkage maps were drawn by JoinMap 3.0 output.</p>
   <h3>QTL mapping</h3>
   <p>Composite Interval Mapping (CIM) analysis was implemented using Windows QTL Cartographer v.2.5 [25] with a 2-cM walk speed and a type-I error rate of 5%. Intervals of five background markers with a window width of 10 cM were analyzed to control the QTL background effects. The Kosambi mapping function was chosen for recalculation of maps on genotype data. The LOD significance threshold was determined by a permutation test with 1000 permutations at a significance level of P < 0.05. The QTL results were drawn using a custom Perl script, which is available upon request.</p>
</body>
</html>
