<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>results</title>
</head>
<body>
   <h1>Results</h1>
   <h3>Construction and sequencing of ddRAD libraries</h3>
   <p>A total of 138 ddRAD libraries were constructed from the two parents (each in 8 replicates) and their 122 F1 offsprings. The 138 ddRAD libraries were sequenced to generate 236.56 million reads, comprising approximately 33.83 Gb of sequencing data (Additional file 2: Figure S3). The female and male parental data sets contained 20.17 million reads (comprising 2.88 Gb) and 13.08 million reads (1.87 Gb), respectively. From the 122 offspring, 204.108 million reads containing 29.07 Gb of data were produced for SNP detection. 82.62% and 81.89% of reads were successfully mapped to the P. trichocarpa reference genome for the female and male parent, respectively. For the 122 progenies the average percentage of mapped reads was 77.51%.</p>
   <h3>SNP genotyping</h3>
   <p>GATK identified 16,234 SNPs to further analysis after passing through filtering criteria (see Methods). Using these SNPs, 122 progenies were screened for their individual ancestry by maximum likelihood estimation using the ADMIXTURE software [26]. The number of ancestral populations was K = 2, corresponding to 8 reps of each female and male parent (Additional file 3: Figure S4). 7 out of 122 F1 offsprings showed unclear ancestry and were excluded from the further analysis, whereas 115 progenies perfectly demonstrated their hybrid nature and were used for the linkage map construction.  Next, chi-square tests were performed for each of 16,234 SNPs to check for correspondence to Mendelian segregation ratios 1:1, 1:2:1 and 1:1:1:1 within the 122 F1 progenies. Those SNPs deviating seriously from the Mendelian ratios (p > 0.05) were excluded from the analysis. There were SNPs showing no polymorphism between the 2 P. tremula parents, but polymorphic when compared to the P. trichocarpa reference. Those SNPs were considered as potentially valuable markers and were included in the final list of 15,732 robust SNPs useful for genotyping purposes in aspen (Additional file 4).</p>
   <h3>High-resolution genetic map construction</h3>
   <p>To construct the female map SNPs from the 15,732 robust SNPs list were selected if they i) were maternal heterozygous (ab) and paternal homozygous (aa), ii) showed segregation ratio 1:1 among the progeny, iii) showed clear polymorphism when 8 reps of maternal genotype and 8 reps of paternal genotype were compared, iiii) had less than 10% of missing data across the entire F1 population. Based on these criteria, 1000 SNPs were chosen for female map construction. Correspondingly, 1055 SNPs that were paternal heterozygous (ab) and maternal homozygous (aa) passing through the above mentioned criteria were selected to develop the male linkage map</p>
   <p>Due to the high level of synteny and collinearity between the linkage maps of different Populus species and physical map of P. trichocarpa, [11, 12, 27] we divided the SNPs into 19 preliminary groups according to their distribution across the 19 chromosomes of the reference genome of P. trichocarpa. SNP markers were then assembled separately for each of 19 chromosomes for the male and female map using default parameters of JoinMap 3.0. For the best mapping order of the SNP markers on a chromosome and precise estimation of the genetic distances, F1 offsprings showing more than 10% of missing data for a particular linkage group were excluded from the mapping procedure. Therefore, the number of F1 individuals involved in SNP mapping for each chromosome varied between 99 to 115. For the resulting maternal map 1000 SNPs were assigned to 19 linkage groups at the LOD threshold of 10.0 comprising a total genetic distance of 3054.99 cM. The length of single LG ranged from 106.01 cM (LG9) to 300.24 cM (LG1), with an average interlocus distance of 3.05 cM (Table 2). The corresponding paternal map represented a total length of 3090.56 cM and ranged from 107.3 cM (LG9) to 361.07 cM (LG1), with an average distance between neighboring markers of 2.93. There was a good consistency between the two parental maps in terms of length for the 19 linkage groups with the total correlation coefficient 0.91. The highest similarity was detected for LG2 and LG9, the deepest discordance was revealed for LG3. </p>
   <p>We also estimated syntenic relationships between the maternal and paternal linkage maps of P. tremula to the physical map of P. trichocarpa (Fig. 3). The order of SNP markers in the aspen parental genetic maps and the P. trichocarpa genome was highly consistent, showing high levels of synteny between the genomes of the 2 Populus species. However, for some linkage groups e.g. Chr 1,11,12,15,17,18 (Fig. 3) a block of SNPs was detected whose genetic distances were not congruent to the respective SNP positions in the chromosomes of P. trichocarpa. Since both maternal and paternal maps showed the same mode of the discordancy relatively to the reference genome, it suggests the presence of chromosome inversions, such as those found on Chr 1, 11, 15, 18.</p>
   <p>Recently published high resolution linkage maps of P. deltoides and P. simonii allowed estimating of correlation between the genetic length of the 19 chromosomes of both the species and physical chromosome size of P. trichocarpa [11]. In the present study we developed the high density genetic maps for P. tremula. Thus, it was worthwhile to estimate the correlation between the chromosome sizes of all four Populus species (Table 3). The highest correlation (0.98) was detected between the genetic map of the P. tremula male parent and the physical map of P. trichocarpa, suggesting that the linkage map is quite accurate in marker ordering. The correlation coefficients between genetic lengths of linkage groups on the male P. tremula map and in the maps of P. deltoides and P. simonii were nearly identical (0.927 and 0.929 respectively).</p>
   <p>The F1 mapping population deriving from the intra-specific cross of two P. tremula genotypes was developed to monitor phenotypic trait variation in aspen under different environmental conditions. To aid in that that, experimental planting of F1 progenies were established in three different geographical locations facilitating further QTL mapping studies. Since in 2017 only one-year-old seedlings of the F1 population were available for phenotype assessment, we estimated the plant height variation of the seedlings before they were planted in the field (Additional file 7: Figure S5). Among the hybrids of the mapping population, the height of seedlings varied extensively from 15 cm to 127 cm, following a normal distribution (p = 0.5450) (Additional file 7: Figure S6). The estimated significant threshold for the seedling height QTLs was determined using a permutation test as LOD = 3.4 (P < 0.05). The most significant QTL (LOD = 5.73) was localized to LG5 (96.94 cM) of the male linkage map, explaining 18% of the phenotypic variation (Fig. 4). The nearest co-segregated markers were NC_008471.2_15136737; NC_008471.2_13107624 and NC_008471.2_12784748. Another significant QTL related to one-year-seedling height variation was identified on LG16 of the female map at 70.84 cM in an interval between SNPs NC_008482.2_6139958 and NC_008482.2_6542789. Its contribution to the phenotypic variation was estimated as 15%.</p>
</body>
</html>
