<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>references</title>
</head>
<body>
   <h1>References</h1>
   <p>1. Yablokov AS. Upbringing and breeding of healthy aspen. Leningrad:
Goslesbumizdat; 1949. In Russian</p>
   <p>2. Garipov NR. Selection and propagation of triploid aspen (Populus tremula L.)
with the application of methods of molecular genetics and biotechnology
in the Republic of Tatarstan. 2014. http://dlib.rsl.ru/01005552669. Accessed
16 Oct 2017.</p>
   <p>3. Tzarev AP. ‘Sortology’ of poplar. Voronezh: Voronezh State University; 1985.
In Russian</p>
   <p>4. Frewen BE, Chen TH, Howe GT, Davis J, Rohde A, Boerjan W, Bradshaw HD.
Quantitative trait loci and candidate gene mapping of bud set and bud
flush in Populus. Genetics. 2000;154(2):837–45.</p>
   <p>5. Luquez V, Hall D, Albrectsen BR, Karlsson J, Ingvarsson P, Jansson S. (2008).
Natural phenological variation in aspen (Populus tremula): the SwAsp
collection. Tree Genetics Genomes. 2008;4(2):279–92.</p>
   <p>6. Tuskan GA, Difazio S, Jansson S, Bohlmann J, Grigoriev I, Hellsten U, et al.
The genome of black cottonwood, Populus trichocarpa (Torr. & gray).
Science. 2006;313:1596–604. PMID: 16973872</p>
   <p>7. Geraldes A, Difazio SP, Slavov GT, Ranjan P, Muchero W, Hannemann J, et al.
A 34K SNP genotyping array for Populus Trichocarpa: design, application to
the study of natural populations and transferability to other Populus species.
Mol Ecol Resour. 2013;13(2):306–23.</p>
   <p>8. Wang Y, Sun X, Tan B, Zhang B, Xu LA, Huang M, Wang M. A genetic
linkage map of Populus adenopoda maxim. × P. alba L. hybrid based on SSR
and SRAP markers. Euphytica. 2010;173(2):193–205.</p>
   <p>9. Pakull B, Groppe K, Mecucci F, Gaudet M, Sabatti M, Fladung M. Genetic
mapping of linkage group XIX and identification of sex-linked SSR markers
in a Populus tremula× Populus tremuloides cross. Can J For Res.
2011;41(2):245–53.</p>
   <p>10. Pakull B, Kersten B, Lüneburg J, Fladung M. A simple PCR-based marker to
determine sex in aspen. Plant Biol. 2015;17(1):256–61.</p>
   <p>11. Tong C, Li H, Wang Y, Li X, Ou J, Wang D, et al. Construction of high-density
linkage maps of Populus deltoids × P. simonii using restriction-site associated
DNA sequencing. PLoS One. 2016;11(3):e0150692.</p>
   <p>12. Mousavi M, Tong C, Liu F, Tao S, Wu J, Li H, Shi J. De novo SNP discovery and
genetic linkage mapping in poplar using restriction site associated DNA and
whole-genome sequencing technologies. BMC Genomics. 2016;17(1):656.</p>
   <p>13. Michelson IH, Ingvarsson PK, Robinson KM, Edlund E, Eriksson ME, Nilsson O,
Jansson S. Autumn senescence in aspen is not triggered by day length.
Physiol Plantarum. 2017. doi:10.1111/ppl.12593.</p>
   <p>14. Peterson BK, Weber JN, Kay EH, Fisher HS, Hoekstra HE. Double digest
RADseq: an inexpensive method for de novo SNP discovery and
genotyping in model and non-model species. PLoS One. 2012;7(5):e37135.</p>
   <p>15. Sun C, Niu Y, Ye X, Dong J, Hu W, Zeng Q, et al. Construction of a high-
density linkage map and mapping of sex determination and growth-related
loci in the mandarin fish (Siniperca chuatsi). BMC Genomics. 2017;18(1):446.</p>
   <p>16. Li JT, Yang J, Chen DC, Zhang XL, Tang ZS. An optimized mini-preparation
method to obtain high-quality genomic DNA from mature leaves of
sunflower. Genet Mol Res. 2007;6(4):1064–71.</p>
   <p>17. Plekhanova E, Vishnyakova MA, Bulyntsev S, et al. Genomic and phenotypic
analysis of Vavilov’s historic landraces reveals the impact of environment
and genomic islands of agronomic traits. Scientific Reports. 2017;7:4816. doi:
10.1038/s41598-017-05087-5.</p>
   <p>18. Li H, Durbin R. Fast and accurate long-read alignment with burrows–
wheeler transform. Bioinformatics. 2010;26(5):589–95.</p>
   <p>19. McKenna A, Hanna M, Banks E, Sivachenko A, Cibulskis K, Kernytsky A, et al.
The genome analysis toolkit: a MapReduce framework for analyzing next-
generation DNA sequencing data. Genome Res. 2010;20(9):1297–303.</p>
   <p>20. DePristo MA, Banks E, Poplin R, Garimella KV, Maguire JR, Hartl C, et al. A
framework for variation discovery and genotyping using next-generation
DNA sequencing data. Nat Genet. 2011;43(5):491–8.</p>
   <p>21. Van der Auwera GA, Carneiro MO, Hartl C, Poplin R, del Angel G, et al. From
FastQ data to high-confidence variant calls: the genome analysis toolkit
best practices pipeline. Curr Protoc Bioinformatics. 2013;11:10.</p>
   <p>22. Danecek P, Auton A, Abecasis G, Albers CA, Banks E, DePristo MA, et al. The
variant call format and VCFtools. Bioinformatics. 2011;27(15):2156–8.</p>
   <p>23. Grattapaglia D, Sederoff R. Genetic linkage maps of Eucalyptus grandis and
Eucalyptus urophylla using a pseudo-testcross: mapping strategy and RAPD
markers. Genetics. 1994;137(4):1121–37.</p>
   <p>24. Van Ooijen, Johan W, Voorrips RE. "JoinMap® 3.0, Software for the
calculation of genetic linkage maps." Plant Research International,
Wageningen, The Netherlands. (2001);1–51.</p>
   <p>25. Wang S, Basten CJ, and Zeng Z-B. Windows QTL Cartographer 2.5.
Department of Statistics, North Carolina State University, Raleigh, NC. 2012.
http://statgen.ncsu.edu/qtlcart/WQTLCart.htm. Accessed 01 Aug 2012.</p>
   <p>26. Alexander DH, Novembre J, Lange K. Fast model-based estimation of
ancestry in unrelated individuals. Genome Res. 2009;19:1655–64.</p>
   <p>27. Wang Y, Zhang B, Sun X, Tan B, Xu LA, Huang M, et al. Comparative
genome mapping among Populus adenopoda, P. alba, P. deltoides, P.
euramericana and P. trichocarpa. Genes Genet Syst. 2011;86(4):257–68.</p>
   <p>28. Ingvarsson PK. Nucleotide polymorphism and linkage disequilibrium within
and among natural populations of European aspen (Populus tremula L.,
Salicaceae). Genetics. 2005;169(2):945–53.</p>
   <p>29. Baird NA, Etter PD, Atwood TS, Currey MC, Shiver AL, Lewis ZA, et al. Rapid
SNP discovery and genetic mapping using sequenced RAD markers.
PLoS One. 2008;3(10):e3376.</p>
   <p>30. Kunitskii B. Botanical and silvicultural characteristics of aspen with notes on
its use. Yearbook of St.Petersburg Forest Institute. 1888;2:51–71.</p>
   <p>31. Rubtzov NI. A note about a new form of aspen from the Trans-Ili Alatau.
Botanical Journal of the USSR. 1938;23(1):23–26.</p>
   <p>32. Kruglikov GG. On a new form of aspen. Forestry. 1938;6(12):34–42.</p>
   <p>33. Kirkpatrick M. How and why chromosome inversions evolve. PLoS Biol.
2010;8(9):e1000501. doi:10.1371/journal.pbio.1000501.</p>
   <p>34. Lowry DB, Willis JH. A widespread chromosomal inversion polymorphism
contributes to a major life-history transition, local adaptation, and
reproductive isolation. PLoS Biol. 2010;8:e1000500. doi:10.1371/journal.pbio.
1000500.</p>
</body>
</html>
