<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>background</title>
</head>
<body>
   <h1>Background</h1>
   <p>Populus species are the fast-growing trees appreciated worldwide due to their rapid growth and quality of wood, which is amenable to a variety of mechanical and chemical processing. They are of particular importance for the pulp and paper industries and the fuel and energy sectors, providing valuable raw materials in a short time. In Europe, the most frost-resistant and productive Populus species that has adapted to unfertile and acidic soils is aspen (Populus tremula L.) [1]. Despite its value and advantages, aspen has a major drawback - massive trunk damage by heart rot caused by the fungi Phellinus tremulae (Bond.) Bond. et. Boriss. Aspen clones with the highest growth rate (i.e. natural triploids) have been reportedly less affected by the heart rot pathogen [1, 2]. Therefore, the priority tasks of the aspen breeding programs in Russia have been growth rate, resistance to diseases and winter hardiness, which in turn, are closely related to the phenology traits [3].</p>	
   <p>The timing of bud flush and bud set are important adaptive traits which represent critical trade-offs between survival and growth [4]. Too early bud flush or late bud set increases the risk of frost damage, whereas too early bud set might be disadvantageous in the competition for resources with other trees during the short growing season characteristic of northern latitudes [5]. The knowledge about the genetic architecture of the adaptive and grow-related traits will allow effective breeding of the most valuable commercial aspen clones.</p>
   <p>Breeding in woody perennial species is a long-term process where the availability and application of genomic tools would have obvious benefits. Several remarkable genomic resources for Populus breeding were recently made available, e.g. the assembled genome sequence of a P. trichocarpa [6], a 34 K SNP array available for genotyping of natural populations of Populus trichocarpa and other Populus species [7], and a number of genetic linkage maps of different species in the genus (reviewed in [8]). The linkage maps allowed identification of QTLs related to valuable traits followed by candidate gene identification and developing gene-specific markers that should facilitate breeding in long-lived perennial plants like Populus. For example, the linkage maps developed for full-sib F1 progenies of interspecific crosses between P. tremula x P. tremuloides allowed identification of QTLs related to sex determination in aspen [9]. Consequently, the sex-linked genomic region was reviewed considering gene loci located in congruent genome region of the sequenced P. trichocarpa. These genes were bioinformatically evaluated for their potential as candidate genes for sex determination in aspen, suggesting as a target Potri.019G047300 (‘TOZ19’). A PCR-based marker was subsequently developed for determination of the sex of non-flowering aspen individuals or seedlings [10].</p>
   <p>Technologies such as next-generation sequencing (NGS) provide new inspiring opportunities to obtain genome-wide genetic markers for high resolution linkage map construction. Recently, Restriction-site Associated DNA Sequencing (RADseq) was employed to identify a large number of SNPs for linkage mapping of F1 progenies derived from an interspecific cross of North American P. deltoides and P. simonii, a native tree species widely distributed in northern China. 2545 high quality SNP markers allowed construction of two parent-specific linkage maps with an average distance of ~3 cM between adjacent markers, facilitating extensive QTL mapping studies in poplar [11]. Using the same F1 hybrid population of P. deltoides × P. simonii Mousavi et al. [12] demonstrated a strategy for constructing dense genetic linkage maps in hybrid forest trees by combining RADseq and whole-genome sequencing technologies.</p>
   <p>So far only inter-specific Populus crosses were assessed for the high-resolution linkage mapping, although some Populus species with huge distribution area would provide the large diversity of phenotypic traits that is comparable with variability observed among the different species [3]. In Europe two native Populus species have a sufficiently wide geographical distribution to be considered for studies of natural variation: black poplar (P. nigra) and aspen (P. tremula) [5].</p>
   <p>The distribution area of P. tremula covers Western and Eastern Europe, the European part of Russia, the Crimea, the Caucasus, Siberia, the Far East, Kazakhstan, Central Asia, Mongolia, China, the northern part of the Korean peninsula and Algeria [1]. Inhabiting a wide range of environmental conditions, aspen exhibits variability in numerous morphological and physiological traits. To analyze the natural phenotypic and genetic variation, the Swedish Aspen (SwAsp) Collection was established to reveal genetic determinants underlying genotypic variation of traits important for adaptation [5]. Investigation of the aspen germplasm collection elucidated exceptionally high levels of phenotypic and genotypic variation in aspen and exposed high levels (>0.55) of broad-sense heritability of phenology traits such as bud set, bud burst, and onset of senescence. Remarkably, a significant correlation between bud set and growth was reported [13]. The availability of a comprehensive set of SNP markers with known position on the aspen linkage map would facilitate the future identification of genes controlling adaptive traits.</p>
   <p>The objective of this study was to construct high-density genetic linkage maps for the European aspen (P. tremula) using a comprehensive set of SNP markers generated with RADseq data. Herein, double digest RADseq (ddRADseq) method [14] was used for the genotyping purposes. The ddRADseq technique was earlier reported as an efficient and low-cost method for de novo SNP discovery and genotyping of a non-model species compared to the existing RADseq approaches [14, 15]. The linkage maps would provide a tool for identifying quantitative trait loci (QTLs) underlying the variation of forestry-important biological traits such as fast growth, resistance to heart rot, wood quality, root-cropping ability, phenology and winter hardiness.</p>
</body>
</html>
